### DISCUSSION TITLE
Issue with character not animating correctly after modifying any of the character BP properties

### DISCUSSION THREAD

**Discussion by LilSaints** [Sixze/ALS-Refactored#221] [General]
Hey there, so I have a custom skeletal mesh which I successfully retargeted (duplicated animBP,  montages etc ). Everything went super smooth until I start changing the data assets in the character blueprint , then the character legs stop animating. Took a while to figure out that doesn't matter what I change in the character, as soon as I swap a data asset for another the legs stop animating. 

Here's an example of what happens to the character when I try swapping the settings asset (even if I swap a single asset in the select mantle settings function the same thing happens).

https://youtu.be/4LePiMk1Eos

**Comment by ameaninglessname** [comment]
I am not able to reproduce it, could you provide more details about steps to reproduce?

**Reply by LilSaints** [reply]
Hey, so what I have done is:
I subclassed AlsCharacter but have the same code as in AlsCharacterExample
Retargeted the animman AnimBP (I chose to retarget all the instances and the main one, not the monolithic) to our custom character.
I'll try to reproduce this in a clean project and share a link if that still happens

**Reply by LilSaints** [reply]
I tested by just replacing the skeletal mesh and animBP in B_Als_Character and I'm still encountering the same issue

**Reply by LilSaints** [reply]
Okay, after migrating the data into a fresh project the problem seemed to be permanent, not only after switching assets (probably due to Perforce ?). Now that I got it narrowed down I can see it's ik related.

**Reply by LilSaints** [reply]
By disconnecting the ControlRig in the anim BP everything goes back to normal , I'm wondering if the bone transform nodes require that the skeleton bone names are not using any capital letters. Anything I should be looking at to fix the issue rather than bypassing the CR?
Thanks
P.s I spotted this old sequence node in the CR
![sequence](https://user-images.githubusercontent.com/74615203/211364916-d4c641ee-3391-4aa9-b5f5-a7f05a616ff6.png)
